cli/src/pcluster/api/controllers/cluster_compute_fleet_controller.py

```diff
@@ -49,6 +60,37 @@ def update_compute_fleet_status(cluster_name, update_compute_fleet_status_reques
 
     :rtype: None
     """
-    update_compute_fleet_status_request_content = UpdateComputeFleetStatusRequestContent.from_dict(
-        update_compute_fleet_status_request_content
-    )
+    try:
+        update_compute_fleet_status_request_content = UpdateComputeFleetStatusRequestContent.from_dict(
+            update_compute_fleet_status_request_content
+        )
+        cluster = Cluster(cluster_name)
+        if not check_cluster_version(cluster, exact_match=True):
+            raise BadRequestException(
+                f"the update can be performed only with the same ParallelCluster version ({cluster.stack.version}) "
+                "used to create the cluster."
+            )
+        status = update_compute_fleet_status_request_content.status
+        if cluster.stack.scheduler == "slurm":
+            if status == RequestedComputeFleetStatus.START_REQUESTED:
+                cluster.start()
+            elif status == RequestedComputeFleetStatus.STOP_REQUESTED:
+                cluster.stop()
+            else:
+                raise BadRequestException(
+                    f"the update compute fleet to status {status} cannot be performed for Slurm clusters."
```

Comment:
why not saying that the compute fleet status can only be set to START_REQUESTED and STOP_REQUESTED?

Also is this covered by tests?